libs/backend/src/websocket/add-player-to-active-players.ts

```diff
@@ -0,0 +1,11 @@
+import { WebSocket } from "@fastify/websocket";
+
+export function addPlayerToActivePlayers({
```

Comment:
honestly, not sure if the `activePlayers` needs to be specified, `addPlayerToPlayers` says just as much, and players have to be active in order to be in there, so it's kind of self explanatory, kind of a pleonasm here

Reply:
nice word => pleonasm => i'll try to remember it

![noted](https://cdn.7tv.app/emote/649fc62a5763d8fb6c1cc074/4x.webp)